src/Hangfire.PostgreSql/PostgreSqlConnection.cs

```diff
@@ -527,7 +527,10 @@ public override long GetCounter(string key)
         throw new ArgumentNullException(nameof(key));
       }
 
-      string query = $@"SELECT SUM(""value"") FROM ""{_options.SchemaName}"".""counter"" WHERE ""key"" = @Key";
+      string query = $@"SELECT SUM(""VALUE"") FROM (
+            SELECT SUM(""value"") VALUE FROM ""{_options.SchemaName}"".""counter"" WHERE ""key"" = @Key
```

Comment:
Please change to `SELECT SUM(""value"") ""value"" FROM ""{_options.SchemaName}"".""counter"" WHERE ""key"" = @Key`